client/src/catalog-element.html

```diff
@@ -707,7 +707,15 @@ <h2 title>[[subElement]]</h2>
         }
       },
 
-      _demoClicked: function() {
+      _demoClicked: function(event) {
+        var isMac = navigator.userAgent.indexOf('Mac OS X') != -1;
+        var mouseEvent = event.detail.sourceEvent;
+        if ((!isMac && mouseEvent.ctrlKey) || (isMac && mouseEvent.metaKey)) {
```

Comment:
I think if ctrl/meta/anything is pressed we should just fall back to the default behavior. Seems less fragile than relying on the useragent.